content/ember-5-0-released.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Ember 5 0 Released
+image:
+imageMeta:
+  attribution:
+  attributionLink:
+featured: true
+authors:
+  - jen-weber
+date: Mon May 15 2023 14:29:48 GMT-0400 (Eastern Daylight Time)
+tags:
+  - releases
+  - '2023'
+  - version-5-x
+---
+
+Today the Ember project is releasing version 5.0 of Ember.js, Ember Data, and Ember CLI. This post will cover our release strategy, how to upgrade, what to expect, and deprecated APIs that were removed.
+
+<!-- TODO - need a nice catchy intro that newsletters can copy paste -->
+
+This is the first major release under the new release strategy and goals proposed in [RFC 830, Evolving Ember's Major Version Process](https://rfcs.emberjs.com/id/0830-evolving-embers-major-version-process). 
+To summarize, every 12 minor release, Ember will ship a new major release, which removes any deprecated code targeted for that major version. 
+When it comes to introducing new features, Ember generally aims to ship new features in minor releases, offering backwards compatibility for existing code at the same time as giving developers the change to try out new capabilities.
+This approach reduces the challenges that teams face for major upgrades, compared to producing big, breaking, splashy major versions with lots of new features.
```

Comment:
A thought (and I'll see if I can whip up some prose to this effect next week): it might be great to expand on this by not just saying it, but by actually describing some of the work we did in the 4.x era, including TS support and getting the `<template>` feature actually out in the world (even if not yet "recommended"). That will make the "Nothing to see here!" vibe a lot milder: instead it will be "Here are the things we shipped between major releases, *so that* major release can be not-a-big-deal-just-some-cleanup."

Reply:
That is a fantastic idea! That would really make the point more tangible and interesting.